ansible/roles/operator-pipeline/templates/openshift/tasks/query-publishing-checklist.yml

```diff
@@ -32,4 +35,5 @@ spec:
 
         hydra-checklist \
           --cert-project-id "$(params.cert_project_id)" \
-          --hydra-url $HYDRA_URL
+          --hydra-url $HYDRA_URL \
+          --ignore-publishing-checklist "$(params.ignore_publishing_checklist)"
```

Comment:
Is this really necessary to pass the flag to the python script? If we want to just ignore a checklist why don't you just not call the python script? 
By skipping the python script you don't have to do any changes in the script at all.

Reply:
I requested this in the acceptance criteria.

> The checklist should still be queried to ensure the configuration is correct.